recipes/scotch/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd src/
+echo 'prefix = $PREFIX' > Makefile.inc
+echo '' >> Makefile.inc
+if [ `uname` == "Darwin" ]; then
+    cp Make.inc/Makefile.inc.i686_mac_darwin10 Makefile.inc
+    sed -i '' 's/-DSCOTCH_PTHREAD//g' Makefile.inc
+    sed -i '' 's/-O3/-O3 -fPIC/g' Makefile.inc
+else
+  cp Make.inc/Makefile.inc.x86-64_pc_linux2 Makefile.inc
+  sed -i "s@CFLAGS\t\t=@CFLAGS\t= -I${PREFIX}/include@" Makefile.inc
```

Comment:
So I looked at the source briefly. They don't have a configure. 😖

My recommendation would be to copy these files into the recipe and modify them as you see fit. We did this with [SuiteSparse]( https://github.com/conda-forge/suitesparse-feedstock/tree/41f46eef1e79839154a7211460a883b3478228fa/recipe ) and had much success. Keeping this `sed` stuff is more likely to cause you hurt later on.